Sources/SwiftDocC/Infrastructure/Diagnostics/DiagnosticFileWriter.swift

```diff
@@ -0,0 +1,30 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2023 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See https://swift.org/LICENSE.txt for license information
+ See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+import Foundation
+
+public final class DiagnosticFileWriter: DiagnosticConsumer {
```

Comment:
The public APIs here could benefit from some documentation comments

Reply:
Fixed in https://github.com/apple/swift-docc/pull/494/commits/0d08bf690f8fac3d462e0a739028ec34493bd6a9